ramoun/chapter_02/ex2_3.cpp

```diff
@@ -0,0 +1,14 @@
+// ex2_3.cpp
+// arith-assignment and decrement ops
+#include <iostream>
+
+using namespace std;
+
+int main() {
```

Comment:
### better be consistent with coding style

- braces on the same level as code level
```cpp
int main () 
    {
    // code level
    }
```
- braces on their own level:
```cpp
int main() {
    // code level
}
```